tests/unit_tests/linear_solver/CMakeLists.txt

```diff
@@ -55,6 +55,8 @@ COMMAND ${MPIGO} $<TARGET_FILE:Tests.exe> nIter)
 ADD_TEST(NAME NNLS_max_iter
 COMMAND ${MPIGO} $<TARGET_FILE:Tests.exe> maxIter)
 
+ADD_TEST(NAME NNLS_multi_core
+COMMAND /usr/bin/mpiexec -np 2 $<TARGET_FILE:Tests.exe> multiCore)
```

Comment:
Is there a reason why you use `/usr/bin/mpiexec -np 2` rather than `mpirun -n ${MPIMAX} ` ?

Reply:
There was not, I just forgot to change it in the last commit. 
I have changed it to `mpirun -n ${MPIMAX}` in the latest commit

Reply:
Great. Just wondering, if the user is running a lot of cores and there are more cores than the length of the vector in the unit test, would that cause an issue?

Reply:
When a user runs the NNLS with more cores than the length of the vector, some of the Epetra_Vector data stored on a core would just have 0 local elements. It does not cause an issue inside the NNLS solver, as all of the solve is done on the root anyway.